web/src/pages/mgmt/modifyBusiness.tsx

```diff
@@ -120,7 +118,7 @@ const ModifyBusinessPage = (): ReactElement => {
                     onChange={(event) => setLicenseStatus(event.target.value)}
                     variant="outlined"
                   >
-                    {licenseStatuses.map((licenseStatus) => (
+                    {licenseStatuses.sort().map((licenseStatus) => (
```

Comment:
Pebble: Consider moving the sort() operation outside the JSX to avoid redundant sorting on each render. 

```


const sortedLicenseStatuses = [...licenseStatuses].sort();
...
   {sortedLicenseStatuses.map((licenseStatus) => (
                      <MenuItem key={licenseStatus} value={licenseStatus}>
                        {licenseStatus}
                      </MenuItem>
                    ))}

```